doc/source/guide/in_depth/operator_guide.rst

```diff
@@ -28,15 +28,12 @@ operator
 
 for a matrix :math:`A\in \mathbb{R}^{n\times m}` as follows::
 
-    from builtins import super
-    import numpy as np
-
     class MatVecOperator(odl.Operator):
         def __init__(self, matrix):
             self.matrix = matrix
             dom = odl.rn(matrix.shape[1])
             ran = odl.rn(matrix.shape[0])
-            super().__init__(dom, ran)
+            odl.Operator.__init__(self, dom, ran)
```

Comment:
Frankly to make the example more simple. Python 2/3 compatibility makes it harder to understand for absolute beginners who may not even know there is a difference.